packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from 'react-apollo';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { FetchAllMenuFeatureToggles } from './fetch-all-menu-feature-toggles.proxy.graphql';
```

Comment:
Don't forget to pull the schema before merging

Reply:
Yup. Just need it deployed :). Will continue next week.

Reply:
Did you pull the schema?

Reply:
![CleanShot 2019-11-11 at 12 54 55@2x](https://user-images.githubusercontent.com/1877073/68585424-81503b80-0482-11ea-995c-e8608e0da031.png)